components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -19,16 +19,14 @@
 #import "MaterialTypography.h"
 #import "supplemental/MDCBannerView.h"
 
-static const CGFloat exampleListTableViewHeight = 300.0f;
-static const CGFloat exampleBannerContentMultiLineWidth = 320.0f;
-static const CGFloat exampleBannerContentSingleLineWidth = 350.0f;
+static const CGFloat exampleListTableViewHeight = 160.0f;
```

Comment:
Please remove these example changes - they should not be required for internal component changes like fixing RTL support.